slider/app.js

```diff
@@ -43,18 +58,21 @@ const applySliderConfigs = function (color, size, range, defaultPosition) {
         containerHeight = "500px";
     }
     sliderContainer.style.height = containerHeight;
+    sliderConfigs.sliderSize = size;
+}
 
-    // Set the Range of the Slider
+const applySliderConfigs = function (color, size, range, defaultPosition) {
+    setKnobColor(color);
+    setSliderSize(size);
     writeSliderRange(range);
-
-    //set default position of Slider Knob (max, middle, min)
     setDefaultPositon(defaultPosition);
 }
 
-const rangeArray1 = [0, 20, 40, 60, 80, 100];
-const rangeArray2 = [0, 10, 20, 30, 40, 50, 60, 70, 80, 90, 100];
+// Default UI settings
+applySliderConfigs(sliderConfigs.knobColor, sliderConfigs.sliderSize, sliderConfigs.sliderRange, sliderConfigs.knobPosition);
```

Comment:
you can pass in sliderConfigs object, you don't need to pass in each argument from it separately. 
like this, as 1 argument: applySliderConfigs(sliderConfigs)